fe/fe-core/src/main/java/org/apache/doris/nereids/properties/RequestPropertyDeriver.java

```diff
@@ -142,7 +142,7 @@ public Void visitPhysicalHiveTableSink(PhysicalHiveTableSink<? extends Plan> hiv
 
     @Override
     public Void visitPhysicalResultSink(PhysicalResultSink<? extends Plan> physicalResultSink, PlanContext context) {
-        addRequestPropertyToChildren(PhysicalProperties.GATHER);
+        addRequestPropertyToChildren(PhysicalProperties.ANY);
```

Comment:
If `order by` exists in this query ,we need a `GATHER` exchange opeartor to do global merge sort here.